### PR TITLE
docs(core-features): migrate Android SplashScreen

### PR DESCRIPTION
**Pull request by erisu**
### Motivation, Context & Description

Move Splash Screen documentation for iOS and Android out of the Plugin section as it has become part of the platform Core code.

### CODE REVIEW

File: www/docs/en/dev/core/features/splashscreen/index.md [hunk 1]
```diff
@@ -0,0 +1,597 @@
...
+### `AndroidWindowSplashScreenBrandingImage`
+
+> :warning: this setting is currently unsupported by the Android's Splash Screen compatability library. [See Google Android Issue Tracker](https://issuetracker.google.com/issues/194301890)
+
+<!--
+The splash screen's branding image.
+
+**Supported Platforms:**
+
+- Android
```
Comment (jcesarmobile) [inline www/docs/en/dev/core/features/splashscreen/index.md:498]: The warning says "this setting is currently unsupported by the Android's Splash Screen compatibility library.", but I don't think people will know what that means, so maybe we should add here "Android 12+"?
Or edit the warning to include that the compatibility library is used on Android <= 11?

Comment (erisu) [reply www/docs/en/dev/core/features/splashscreen/index.md:498]: For the target release and what will be supported in 11.0.0, I updated the message as such:

> > :warning: this setting is currently unsupported. The core-splashscreen compatibility library, that Cordova Android relies on for providing backwards support, has not implemented this functionality. [See Google Android Issue Tracker](https://issuetracker.google.com/issues/194301890)

How is this?

**Comment by erisu** [comment]
I temporarily converted this PR as a draft PR.

I would like to review and merge in #1252 first, which migrates the iOS Splash Screen docs. iOS Splash Screen was already integrated as a core feature of the platform while Android is currently in the release process.

File: www/docs/en/11.x/core/features/splashscreen/index.md [hunk 2]
```diff
@@ -351,12 +391,16 @@ Amount of time in milliseconds to wait before automatically hide splash screen.
...
 
+- Android
 - iOS
 
 **Data Type:** `Number`, in milliseconds
 
 **Default Value:** `true`
 
+- Android
+  `-1`: The splash screen will automatticly hide when the `onPageFinished` has been triggered.
```
Comment (breautek) [inline www/docs/en/11.x/core/features/splashscreen/index.md:402]:
```suggestion
  `-1`: The splash screen will automatically hide when the `onPageFinished` has been triggered.
```

File: www/docs/en/11.x/core/features/splashscreen/index.md [hunk 3]
```diff
@@ -351,12 +391,16 @@ Amount of time in milliseconds to wait before automatically hide splash screen.
...
+
 - iOS
   `3000`: The splash screen will automatticly hide in 3 seconds.
```
Comment (breautek) [inline www/docs/en/11.x/core/features/splashscreen/index.md:405]:
```suggestion
  `3000`: The splash screen will automatically hide in 3 seconds.
```

I know this wasn't changed in this PR, but has the same spelling mistake, so may as well correct it at the same time.

File: www/docs/en/dev/core/features/splashscreen/index.md [hunk 4]
```diff
@@ -351,12 +391,16 @@ Amount of time in milliseconds to wait before automatically hide splash screen.
...
 
+- Android
 - iOS
 
 **Data Type:** `Number`, in milliseconds
 
 **Default Value:** `true`
 
+- Android
+  `-1`: The splash screen will automatticly hide when the `onPageFinished` has been triggered.
```
Comment (breautek) [inline www/docs/en/dev/core/features/splashscreen/index.md:402]:
```suggestion
  `-1`: The splash screen will automatically hide when the `onPageFinished` has been triggered.
```

File: www/docs/en/dev/core/features/splashscreen/index.md [hunk 5]
```diff
@@ -351,12 +391,16 @@ Amount of time in milliseconds to wait before automatically hide splash screen.
...
+
 - iOS
   `3000`: The splash screen will automatticly hide in 3 seconds.
```
Comment (breautek) [inline www/docs/en/dev/core/features/splashscreen/index.md:405]:
```suggestion
  `3000`: The splash screen will automatically hide in 3 seconds.
```

File: www/docs/en/dev/core/features/splashscreen/index.md [hunk 6]
```diff
@@ -366,6 +410,120 @@ Amount of time in milliseconds to wait before automatically hide splash screen.
...
+
+**Usage Example:**
+
+```xml
+<preference name="AndroidPostSplashScreenTheme" value="@style/Theme.AppCompat.NoActionBar"/>
+```
+
+### `AndroidWindowSplashScreenAnimatedIcon`
+
+The splash screen image. This preference is used for both animated and non-animated icons. The current acceptable asset files can either be an XML Vector Drwable or PNG.
```
Comment (breautek) [inline www/docs/en/dev/core/features/splashscreen/index.md:435]:
```suggestion
The splash screen image. This preference is used for both animated and non-animated icons. The current acceptable asset files can either be a XML Vector Drawable or PNG.
```

File: www/docs/en/dev/core/features/splashscreen/index.md [hunk 7]
```diff
@@ -366,6 +410,120 @@ Amount of time in milliseconds to wait before automatically hide splash screen.
...
+
+**Usage Example:**
+
+```xml
+<preference name="AndroidWindowSplashScreenAnimatedIcon" value="res/screen/android/splashscreen.xml" />
+```
+
+### `AndroidWindowSplashScreenAnimationDuration`
+
+Defines the duration of the icon's animation if an Animated Vector Drwable is supplied as the splash screen image.
```
Comment (breautek) [inline www/docs/en/dev/core/features/splashscreen/index.md:453]:
```suggestion
Defines the duration of the icon's animation if an Animated Vector Drawable is supplied as the splash screen image.
```

**Review by breautek — APPROVED** [review]
lgtm

**Pull request merged**